src/Indigo/Contracts/Holdings/Types.hs

```diff
@@ -117,14 +180,17 @@ instance TypeHasDoc (ML.StorageSkeleton StorageFields) where
 storageNotes :: Notes (ToT Storage)
 storageNotes = NTPair noAnn (ann "ledger") noAnn
```

Comment:
Apparently now you can make use of https://gitlab.com/morley-framework/morley/-/merge_requests/403 here.

Reply:
oh, right, let's :zhdun: for it as well